server/1.3/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM fabric8/java-centos-openjdk11-jdk
```

Comment:
Ah, just seeing you do this for 1.2 and 1.3. The latter should suffice actually?

Reply:
Well, as we have no 1.3 out yet and also due to need to test release process I think it would be good to have it also for 1.2

Reply:
Ok, leaving that to you. We'll likely do another 1.2 release for downstream anyways.